### DISCUSSION TITLE
Welcome to Web3 Together

### DISCUSSION THREAD

**Discussion by samnang** [web3together/web3together#1] [Announcements]
Welcome to the **Web3 Together** community! Here you can connect with other developers and share knowledge about every part of blockchain development, including DeFi, NFTs, DAO, etc... Jump in by starting a new discussion or joining an existing one.

## TODOs
* [x] Please introduce [yourself](https://github.com/web3together/web3together/discussions/2) to other members, so we all know each other more.
* [x] Joined the forum/discussions by clicking `watch` button on github page.

We welcome people from all levels of experience to join us and share experiences with others to grow together. Please note [Code of Conduct](https://github.com/web3together/web3together/blob/main/CODE_OF_CONDUCT.md).

-- Sharing is caring

**Comment by invisal** [comment]
Oh, only including the blockchain? No other topic? How about development in general?

**Reply by samnang** [reply]
In general I thought others might already have it, so that's why I would like focus on small niche of people, but I would like the idea of software development in general especially deep discussions on best practices and tools that could help to grow to be professional developers

Starting a podcast?

**Reply by invisal** [reply]
Not sure. We have Khmer Coder group in Facebook, but recently I have no time to maintain it.

**Reply by samnang** [reply]
Exactly, but I thought doing a podcast might be easier because we just discuss the topic from our own experiences.
